### PR TITLE
Document PerformanceMark/PerformanceMeasure.detail

### PR DESCRIPTION
**Pull request by Elchi3**
### Description

This PR adds details! 🥁💤 

### Motivation

https://github.com/openwebdocs/project/issues/62

### Additional details

Spec: https://w3c.github.io/user-timing/#performancemark & https://w3c.github.io/user-timing/#performancemeasure

### Related issues and pull requests

Will submit a BCD PR for the mdn_urls later.

### CODE REVIEW

File: files/en-us/web/api/performancemeasure/detail/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Value
+
+Returns the value it is set to (from `markOptions` of {{domxref("Performance.measure","performance.measure()")}}).
+
+## Examples
+
+The following example shows the use of the `detail` property.
+
+```js
+performance.measure("dog", {detail: "labrador", start: 0, end: 12345 });
```
Comment (Josh-Cena) [inline files/en-us/web/api/performancemeasure/detail/index.md:26]:
```suggestion
performance.measure("dog", { detail: "labrador", start: 0, end: 12345 });
```

File: files/en-us/web/api/performancemark/detail/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - API
+  - Property
+  - Reference
+  - Web Performance
+browser-compat: api.PerformanceMark.detail
+---
+
+{{APIRef}}
+
+The read-only **`detail`** property returns arbitrary metadata that has been included in the mark upon construction (either when using {{domxref("Performance.mark","performance.mark()")}} or the {{domxref("PerformanceMark.PerformanceMark","PerformanceMark()")}} constructor).
```
Comment (jpmedley) [inline files/en-us/web/api/performancemark/detail/index.md:15]:
```suggestion
The read-only **`detail`** property returns arbitrary metadata that was included in the mark upon construction (either when using {{domxref("Performance.mark","performance.mark()")}} or the {{domxref("PerformanceMark.PerformanceMark","PerformanceMark()")}} constructor).
```

File: files/en-us/web/api/performancemark/detail/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Value
+
+Returns the value it is set to (from `markOptions` of {{domxref("Performance.mark","performance.mark()")}} or the {{domxref("PerformanceMark.PerformanceMark","PerformanceMark()")}} constructor).
+
+## Examples
+
+The following example shows the use of the `detail` property.
```
Comment (jpmedley) [inline files/en-us/web/api/performancemark/detail/index.md:23]:
```suggestion
The following example demonstrates the `detail` property.
```

File: files/en-us/web/api/performancemeasure/detail/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  - API
+  - Property
+  - Reference
+  - Web Performance
+browser-compat: api.PerformanceMeasure.detail
+---
+
+{{APIRef}}
+
+The read-only **`detail`** property returns arbitrary metadata that has been included in the mark upon construction (when using {{domxref("Performance.measure","performance.measure()")}}.
```
Comment (jpmedley) [inline files/en-us/web/api/performancemeasure/detail/index.md:15]:
```suggestion
The read-only **`detail`** property returns arbitrary metadata that was included in the mark upon construction (when using {{domxref("Performance.measure","performance.measure()")}}.
```

File: files/en-us/web/api/performancemeasure/detail/index.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Value
+
+Returns the value it is set to (from `markOptions` of {{domxref("Performance.measure","performance.measure()")}}).
```
Comment (jpmedley) [inline files/en-us/web/api/performancemeasure/detail/index.md:19]:
```suggestion
Returns the value it was set to (from `markOptions` of {{domxref("Performance.measure","performance.measure()")}}).
```

File: files/en-us/web/api/performancemeasure/detail/index.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+## Examples
+
+The following example shows the use of the `detail` property.
```
Comment (jpmedley) [inline files/en-us/web/api/performancemeasure/detail/index.md:23]:
```suggestion
The following example demostrates the `detail` property.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Looking good. I have a few suggestions though.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**